README.md

```diff
@@ -19,4 +19,5 @@ For more information, please see the links below.
 
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fcodecov%2Fexample-python.svg?type=large)](https://app.fossa.com/projects/git%2Bgithub.com%2Fcodecov%2Fexample-python?ref=badge_large)
 
-test
+test2
```

Comment:
Consider removing this line entirely. The change from 'test' to 'test2' doesn't provide any meaningful information to readers of the README. If this was intended as a test change, it should be removed before merging to maintain the quality of the documentation.